datanode/sqlstore/migrations/0076_teams_stats.sql

```diff
@@ -1,80 +1,79 @@
 -- +goose Up
-CREATE TABLE IF NOT EXISTS teams_stats
-(
-  team_id              BYTEA             NOT NULL,
-  party_id             BYTEA             NOT NULL,
-  at_epoch             BIGINT            NOT NULL,
-  total_volume         NUMERIC(1000, 16) NOT NULL,
-  total_quantum_reward NUMERIC(1000, 16) NOT NULL,
-  games_played         JSONB             NOT NULL,
-  PRIMARY KEY (team_id, party_id, at_epoch)
+CREATE TABLE IF NOT EXISTS teams_stats (
+    team_id BYTEA NOT NULL, party_id BYTEA NOT NULL, at_epoch BIGINT NOT NULL, total_volume NUMERIC(1000, 16) NOT NULL, total_quantum_reward NUMERIC(1000, 16) NOT NULL, games_played JSONB NOT NULL, PRIMARY KEY (team_id, party_id, at_epoch)
```

Comment:
To be fair, that looked more readable as it was initialy.

Reply:
Yeah, my IDE has done it.

Reply:
I will fix